matrix.d.ts

```diff
@@ -965,26 +968,38 @@ export default Matrix;
 
 export class MatrixColumnView extends AbstractMatrix {
   constructor(matrix: AbstractMatrix, column: number);
+  set(rowIndex: number, columnIndex: number, value: number): this;
+  get(rowIndex: number, columnIndex: number): number;
```

Comment:
d.ts generally follows the code itself, and the code does have the seperate get/set definition in every class

This can be bypassed by making an intermendiate not-abstract class, but the definition is better to comply with actual implementation